docs/_docs/maintaining/releasing-a-new-version.md

```diff
@@ -0,0 +1,83 @@
+---
+title: "Releasing a new version"
+---
+
+**This guide is for maintainers.** These special people have **write access** to one or more of Jekyll's repositories and help merge the contributions of others. You may find what is written here interesting, but it’s definitely not for everyone.
+{: .note .info }
+
+The most important thing to understand before making a release is that there's no need to feel nervous. Most things are revertable, and even if you do publish an incomplete gem version, we can always skip that one. Don't hestitate to contact the other maintainers if you feel unsure or don't know what to do next.
+
+### Bump the version
+
+The only important place you need to manually bump the version is in `lib/jekyll/version.rb`. Adjust that, and everything else should work fine.
+
+### Update the history document
+
+Replace the first header of the history document with a version milestone. This looks like the following:
+
+```diff
+-## HEAD
++## 3.7.1 / 2018-01-25
+```
+
+Adjust the version number and the date. Don't worry about removing the `HEAD` part, it'll be regenerated when the time comes.
```

Comment:
“Don’t worry about removing the `HEAD` part” makes me think I should _not_ remove that line, whichever is not the case.

Reply:
Yeah, that's badly worded.